yt/java/ytsaurus-client/src/test/java/tech/ytsaurus/client/rows/SchemaOfEmbeddedEntityTest.java

```diff
@@ -0,0 +1,112 @@
+package tech.ytsaurus.client.rows;
+
+import org.junit.Test;
+import tech.ytsaurus.skiff.SkiffSchema;
+import tech.ytsaurus.skiff.WireType;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+public class SchemaOfEmbeddedEntityTest {
```

Comment:
You should write tests for the following cases:
1. the class has a nested entity (not embedded), and it has an embedded field;
2. in the class there is an embedded field, in which there is a field with a decimal, without specifying the precision and scale in the annotation (in this case, you need to pass the table schema with the specified precision and scale to the create method);
3. integration tests for reading/writing embedded classes (currently only schema construction is implemented, please add the code to EntitySkiffSerializer).

Reply:
1. Isn't this  case already  covered in `SchemaOfEntityTest`
2 and 3 will do, thanks